app/views/layouts/application.html.erb

```diff
@@ -42,6 +42,13 @@
     <% if Octobox.config.push_notifications && logged_in? %>
       <meta name="push_notifications" content="true">
     <% end %>
+    <% if @notifications.to_a.any? %>
+      <% @notifications.each do |notification| %>
+        <% if notification.unread? and !['closed', 'merged'].include?(notification.state) %>
```

Comment:
Would be great to get this added as a method on the Notification class, something like:

```ruby
def prerender?
  unread? and !['closed', 'merged'].include?(state)
end
```